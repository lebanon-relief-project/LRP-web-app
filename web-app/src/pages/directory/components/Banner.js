import React from "react";
import styled from "styled-components";
import devices from "../../../styles/Devices";

const Banner = () => {
  return (
    <>
      <BannerBox>
        <BannerContent>
          <BannerTitle>Directory of Mental Health Experts</BannerTitle>
          <BannerText>
            The DMHE is a service provided by Lebanon Relief Network. In
            Lebanon, there are many dedicated psychotrauma experts and services
            available. Please use one or more of the following fields to search
            for the mental health expert offering the service you require. NB.
            The information is up to date but not updated regularly.
          </BannerText>
          <BannerText>
            NB. The information is up to date but not updated regularly.
          </BannerText>
        </BannerContent>
      </BannerBox>
    </>
  );
};

const BannerBox = styled.div`
  display: flex;
  justify-content: center;
  background-color: #002766;
  flex: 1;
  height: 276px;
  @media (max-width: ${devices.mobile}) {
    height: auto;
  }
  margin-top: 80px;
  color: white;
`;

const BannerContent = styled.div`
  max-width: 960px;
  width: 100%;
  display: flex;
  flex: 1;
  padding-top: 40px;
  padding-bottom: 40px;
  flex-direction: column;

  @media (max-width: ${devices.mobile}) {
    padding: 18px;
    padding-top: 20px;
    padding-bottom: 25px;
  }
`;

const BannerTitle = styled.h1`
  color: white;
  font-size: 38px;
  line-height: 46px;
  font-weight: 900;

  @media (max-width: ${devices.mobile}) {
    font-size: 30px;
    line-height: 38px;
  }
`;

const BannerText = styled.p`
  max-width: 632px;
  width: 100%;
  margin: 0;
  margin-top: 15px;
  padding: 0;
`;

export default Banner;
