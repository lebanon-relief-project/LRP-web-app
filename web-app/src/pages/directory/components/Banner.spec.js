import React from "react";
import { render } from "@testing-library/react";
import "jest-styled-components";
import Banner from "./Banner";

describe("the Banner component", () => {
  it("should match the snapshot", () => {
    const { container } = render(<Banner />);
    expect(container).toMatchSnapshot();
  });
});
