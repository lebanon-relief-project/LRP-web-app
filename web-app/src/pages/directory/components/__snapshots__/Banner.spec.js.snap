// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the Banner component should match the snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  background-color: #002766;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  height: 276px;
  margin-top: 80px;
  color: white;
}

.c1 {
  max-width: 960px;
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  padding-top: 40px;
  padding-bottom: 40px;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c2 {
  color: white;
  font-size: 38px;
  line-height: 46px;
  font-weight: 900;
}

.c3 {
  max-width: 632px;
  width: 100%;
  margin: 0;
  margin-top: 15px;
  padding: 0;
}

@media (max-width:576px) {
  .c0 {
    height: auto;
  }
}

@media (max-width:1025px) {
  .c1 {
    padding: 18px;
    padding-top: 20px;
    padding-bottom: 25px;
  }
}

@media (max-width:576px) {
  .c2 {
    font-size: 30px;
    line-height: 38px;
  }
}

<div>
  <div
    class="c0"
  >
    <div
      class="c1"
    >
      <h1
        class="c2"
      >
        Directory of Mental Health Experts
      </h1>
      <p
        class="c3"
      >
        The DMHE is a service provided by Lebanon Relief Network. In Lebanon, there are many dedicated psychotrauma experts and services available. Please use one or more of the following fields to search for the mental health expert offering the service you require. NB. The information is up to date but not updated regularly.
      </p>
      <p
        class="c3"
      >
        NB. The information is up to date but not updated regularly.
      </p>
    </div>
  </div>
</div>
`;
